frontend/src/config/malawi/index.ts

```diff
@@ -3,7 +3,14 @@ import rawLayers from './layers.json';
 
 const rawTables = {};
 const rawReports = {};
-const translation = {};
+
+// Temp: hack for Malawi window labels
+const translation = {
+  en: {
+    'Window 1': 'NDJ',
```

Comment:
@wadhwamatic can we anticipate having more than 2 windows? or just 1?

Reply:
I don't expect more than two windows, but a single monitoring window is definitely a possibility. Let me double check with Gabriela.

Reply:
@ericboucher - confirmed that we will have a single monitoring window in some countries. No plans for more than 2, and doubtful we would in the future

Reply:
Ok, let's wait until this happens to adapt